src/reducers/reducer.js

```diff
@@ -0,0 +1,19 @@
+const reducer = (state, action) => {
```

Comment:
IMO it's a good practice to define default values for `state` and `action` in a reducer, like this:
```js
const reducer = (state = {}, action = {}) => {
```

Reply:
Resolved.